cicd/cloudbuild_promote_preprod.yaml

```diff
@@ -0,0 +1,41 @@
+steps:
+  - name: gcr.io/$PROJECT_ID/testrunner
+    args:
+      - '-c'
+      - >
+        set -e
+        echo Pulling image dev image with tag $SHORT_SHA ...
+
+        docker pull
+        $LOCATION-docker.pkg.dev/$_DEV_PROJECT_ID/$_GAR_DEV_REPO/$_GAR_PACKAGE:$SHORT_SHA
+
+        echo Tagging pulled image as a release candidate $TAG_NAME ...
+
+        docker tag
+        $LOCATION-docker.pkg.dev/$_DEV_PROJECT_ID/$_GAR_DEV_REPO/$_GAR_PACKAGE:$SHORT_SHA
+        $LOCATION-docker.pkg.dev/$PROJECT_ID/$_GAR_RELEASES_REPO/$_GAR_PACKAGE:$TAG_NAME
+
+        echo Pushing tagged image to release GAR ...
+
+        docker push
+        $LOCATION-docker.pkg.dev/$PROJECT_ID/$_GAR_RELEASES_REPO/$_GAR_PACKAGE:$TAG_NAME
+
+        gcloud run deploy survey-assist-api
+        --image=$LOCATION-docker.pkg.dev/$PROJECT_ID/$_GAR_RELEASES_REPO/$_GAR_PACKAGE:$TAG_NAME
+        --region $LOCATION --project $_PREPROD_PROJECT_ID
+
+        export
+        SURVEY_ASSIST_API_URL="https://survey-assist-api-$_PREPROD_PROJECT_NUMBER.$LOCATION.run.app/v1/survey-assist"
```

Comment:
It might be worth making the API version a build variable at some point. Could you add a ticket to track if it is something we don't want to do now.